Add SketchUp

sjstark opened the pull request:
![sketchup](https://user-images.githubusercontent.com/6759557/93852367-e9b1a100-fc7f-11ea-8263-a634792173b2.png)


**Issue:** #1984
**Alexa rank:** 1,896

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with **SVGO** or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Selected hex value of "block" part of logo as the prominent color feature of the logo.
Logo was taken from [Wikimedia Commons](https://commons.wikimedia.org/wiki/File:SketchUp_logo.svg), but matches [website](https://www.sketchup.com/).

runxel commented on _data/simple-icons.json:
```diff
@@ -5525,6 +5525,11 @@
         },
+        {
+            "title": "SketchUp",
+            "hex": "E9292F",
```
I'm not sure about this color?
Where did you get it from, exactly?

The color of the Wikimedia icon is different, and so is the icon on the website... (`#F62737`) 🤔

runxel reviewed (commented):
Icon looks great, not sure with the color tho.

runxel commented:
Thanks for the work! Looks great! 💯

sjstark commented:
@runxel you're absolutely correct. The color picker software I was using originally apparently awful (or maybe taking into account my f.lux adjustments at the time?)

Either way, I've updated the color in the file to the #F62737 listed.

(I've also updated my color picker to just a chrome extension)

creepertron95 commented:
@sjstark I was kind of curious of what color picker you used. On my computer, I used a lightweight color picker program called [ColorCop](http://colorcop.net/). It's old but it works for me.
![colorcop](https://user-images.githubusercontent.com/53346722/94298575-fd812f80-ff2b-11ea-861f-75432e6a2a08.PNG)

sjstark commented:
The name should have been the first red flag... [Free Color Picker](https://www.microsoft.com/en-us/p/free-color-picker-color-picker-from-screen-html-color-picker-hex-color-picker/9p9qkfj0fh21?activetab=pivot:overviewtab)

I got lazy and picked the first free one I found on the microsoft store. Downloaded the one you linked and like it much more!

runxel commented:
I personally used the ColorCop too for a long time. But it doesn't work with multi displays. So nowadays I just use [PowerToys](https://github.com/microsoft/PowerToys/), which has also a colorpicker, bound to default `win + shift + c` and it's just working.
